deepsampler-provider/src/testFixtures/java/de/ppi/deepsampler/provider/common/PersistentSamplerAspectTest.java

```diff
@@ -544,4 +544,40 @@ private PersistentSampleManager save(Path pathToFile) {
         return source;
     }
 
+    @Test
+    public void polymorphicSamplesCanBeRecordedAndLoaded() throws IOException {
```

Comment:
The extension TempJsonFile injects a Path to a tmp file and deletes that file after the test has run. You can add a parameter `Path tempFile` to the method to get access to the tmp file.

There is also a helper-method that uses the tmpFile to create a PersistnetSampleManager: https://github.com/ppi-ag/deep-sampler/blob/cb9876917ac97d3cc270ba69b3e2e5ee0f39047d/deepsampler-provider/src/testFixtures/java/de/ppi/deepsampler/provider/common/PersistentSamplerAspectTest.java#L541